### ISSUE TITLE
Compilation Error during Make firmware - Probably Ubuntu version mismatch ? workaround inside.

### ISSUE THREAD

**Issue by antoineclaval** [sudomesh/disaster-radio-simulator#16]
Hi,

This is more a question than a issue. 

When I follow the [steps](https://github.com/sudomesh/disaster-radio-simulator#compile-the-firmware) to compile the firmware, I see a compilation error. 

I see the note about running it on Ubuntu 18.04. I have a 20.04, so I gave it a try. 

I ran the prerequisite step, and the ./fetch_deps.sh. It went fine, no error. 

When I do 'make firmware' I see the error below

```
./libs/backward-cpp/backward.hpp:211:10: fatal error: bfd.h: No such file or directory
  211 | #include <bfd.h>
      |          ^~~~~~~
compilation terminated.
make: *** [Makefile:26: firmware] Error 1
```

And then 

```
/usr/bin/ld: cannot find -lboost_filesystem
/usr/bin/ld: cannot find -lboost_system
collect2: error: ld returned 1 exit status
make: *** [Makefile:26: firmware] Error 1
```

I installed additional dependencies to work around that in 20.04. You guys want a documentation MR for that ? 

Long story short : On 20.04, to compile the binary for ./firmware you will need : 

```
apt-get install binutils-dev
apt-get install libboost-filesystem-dev
sudo apt-get install libboost-filesystem-dev
```
